src/libmongoc/src/mongoc/mongoc-server-monitor-private.h

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2020-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "mongoc-prelude.h"
+
+#ifndef MONGOC_SERVER_MONITOR_PRIVATE_H
+#define MONGOC_SERVER_MONITOR_PRIVATE_H
+
+#include "mongoc.h"
+
+/* For background monitoring of a single server. */
```

Comment:
I've split off the logic for monitoring a single server from mongoc-topology-background-monitoring.c into this file.